rmf_traffic/colcon.pkg

```diff
@@ -0,0 +1,6 @@
+{
+  "hooks": [
+    "share/rmf_traffic/hook/ament_prefix_path.dsv", 
+    "share/rmf_traffic/hook/ros_package_path.dsv",
```

Comment:
I think this is primarily used for ROS1? I just added in regardless. I am not too sure.